### PR TITLE
Add 1.13 patch releases info

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Quick draft w/ information about 1.13 patch release schedule proposed in #427. It's not meant to be perfect/include everything - I suspect we may have to iterate on it to find out what's most useful.

/kind documentation
/cc @tpepper @spiffxp

### CODE REVIEW

File: releases/release-1.13/patch_releases.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+branch have to merge to be included in the patch release. Any PRs left pending past
+this point will either wait for the next patch release, or may in rare cases
+delay current release.
+
+Target date is the date on which the patch release is cut. Must be at least 2
+days after cherry-picks deadline to allow all release-blocking tests to run.
+
+| **Patch Release** | **Cherry-picks deadline** | **Target date** | **Owner** |
+| --- | --- | --- | --- |
+| 1.13.2 | TBA | 2019-01-10 | @tpepper |
```
Comment (spiffxp) [inline releases/release-1.13/patch_releases.md]: I'll leave it to you to decide what's appropriate, but I would personally prefer to see a longer forecast.  Something like a suggested cadence, or the next N potential dates, etc.

**Comment by tpepper** [comment]
@feiskyer and @foxish what do you think about having one file, for example releases/patch_releases.md which has a section for each supported branch.  @aleksandra-malinowska 's proposed a 1.13 specific file and information.  I think it would be more easily discovered if the cadence and status info for all the branches was in one place.  Today's set of patch release managers should be able to commit to one file without much conflict.

**Comment by aleksandra-malinowska** [comment]
@tpepper I wanted to add some schedule information instead of getting stuck looking for a perfect solution, but if you believe it's better this way, let's just close this one.

**Comment by feiskyer** [comment]
> what do you think about having one file, for example releases/patch_releases.md which has a section for each supported branch.

It's a good idea to include all in one file. Much easier to find for cherry picks, as well as schedulers.

**Comment by tpepper** [comment]
No need to close this one. If folks prefer one location, just amend your commit to use the one directory higher file location, and at most add one level of indention inside the file with place holder section names for 1.12 and 1.11, and maybe a table of contents at the top of the file. You've done the 1.13 content and there are owners for adding 1.12 and 1.11 content.

**Comment by tpepper** [comment]
I think this is a good start.  Certainly much better than what was visible before.  We can incrementally improve and extend it.

/lgtm

**Pull request merged**